pc/request.cpp

```diff
@@ -708,12 +708,15 @@ bool price::send( price *prices[], const unsigned n )
   for ( unsigned i = 0, j = 0; i < n; ++i ) {
     price *const p = prices[ i ];
     if ( PC_UNLIKELY( ! p->init_ && ! p->init_publish() ) ) {
+      PC_LOG_ERR( "failed to initialize publisher" );
```

Comment:
do these conditions occur naturally, like on startup? might want to make these WARN if so?

Reply:
They were previously logged at error level, and I think it's safer to keep it that way. Having a few spurious errors at statup is better than missing important warnings if something breaks after `pythd` has been running for a while. Open to other opinions though 😄